Lib/booleanOperations/booleanGlyph.py

```diff
@@ -78,7 +82,7 @@ def _get_clockwise(self):
 
     def _get_bounds(self):
         if self._bounds is None:
-            from robofab.pens.boundsPen import BoundsPen
+            from fontTools.pens.boundsPen import BoundsPen
```

Comment:
Yes, the robofab pen ignores single points, or anchors, whereas fonttools one takes them into account. I guess we could make this optional by adding an `ignoreSinglePoints` argument to the constructor of `fontTools.pen.boundsPen.ControlBoundsPen` so that it behaves like robofab's. WDYT?